src/main/java/de/komoot/photon/App.java

```diff
@@ -126,7 +124,7 @@ private static void startNominatimImport(CommandLineArgs args, Server esServer,
         try {
             nominatimConnector.readEntireDatabase();
         } catch (Exception e) {
-            log.info("error importing from nominatim: " + e.getMessage());
+            throw new RuntimeException("error importing from nominatim: " + e.getMessage());
```

Comment:
You can remove the entire try-catch block. readEntireDatabase() only throws RuntimeExceptions in the first place.